numpy/lib/tests/test_histograms.py

```diff
@@ -2,7 +2,7 @@
 
 import numpy as np
 
-from numpy.lib.histograms import histogram, histogramdd
+from numpy.lib.histograms import histogram, histogramdd, histogram_bin_edges
```

Comment:
You should all this to `__all__` in `histograms.py` too.

Reply:
Done.